process/block/baseProcess.go

```diff
@@ -1144,9 +1144,9 @@ func (bp *baseProcessor) commitAll(headerHandler data.HeaderHandler) error {
 }
 
 func (bp *baseProcessor) commitInLastEpoch(currentEpoch uint32) error {
-	lastEpoch := currentEpoch - 1
-	if lastEpoch < 0 {
-		lastEpoch = 0
+	lastEpoch := uint32(0)
+	if currentEpoch > 0 {
```

Comment:
the code was a tiny faster as it was written before as now we only have epochs > 1

Reply:
The linter would have raised a warning for the previous implementation, as lastEpoch (uint32) can never be < 0